src/components/Books.css

```diff
@@ -0,0 +1,86 @@
+.book-card-style {
+  background-color: white;
+  height: 10.625rem;
+}
+
+.title-display-style {
+  margin: 0 20.188rem 0 0;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
```

Comment:
**Changes Required**
I like how you named your css classes, they are descriptive. Well done. However;
- [ ] The `font-style, line-height, letter-spacing` properties with value `normal` and `color` with value `#4386bf`  are over repeated throughout your stylesheet. I suggest you have a class say `.font-style-normal` in your `.jsx` files  and in your `.css` files give it a property of font-style and value of normal. In this way, you will just declare it in only one place and reuse it. It will make your code more readable. Check t[his article on Good and bad practices](https://speckyboy.com/good-bad-css-practices/) in CSS